content/en/internal_developer_portal/software_catalog/entity_model/ai-generated-systems.md

```diff
@@ -0,0 +1,70 @@
+---
+title: AI-generated Systems
+description: Discover recommended System entities powered by Bits AI to organize your Software Catalog.
+disable_toc: false
+further_reading:
+- link: "/internal_developer_portal/software_catalog/entity_model/entity_types"
+  tag: "Documentation"
+  text: "Entity Types"
+- link: "/internal_developer_portal/software_catalog/set_up/discover_entities"
+  tag: "Documentation"
+  text: "Discover Entities"
+---
+
+## Overview
+
+AI-generated Systems surface recommended System entities that you can add to your Software Catalog. Bits AI generates these recommendations using:
+- Observability data, including distributed traces
+- Component dependencies (automatically detected with APM or manually declared)
+- Ownership metadata from your Software Catalog
+
+Recommendations include service, datastore, and queue entity types and automatically populate the following System metadata:
+- Name and display name
+- Owner
+- Description
+- Components
+
+**Note**: AI-generated Systems only suggest new Systems; they do not propose updates to existing ones.
+
+## Enable AI-generated Systems
+
+AI-generated Systems must be enabled once per organization by a user with `org_management` permissions (included in the default [Datadog Admin role][1]).
+
+To enable:
+
+1. Navigate to **IDP** > **Software Catalog** > **Suggestions**.
```

Comment:
Could consider linking to https://app.datadoghq.com/software?selectedComponent=suggestionsTable here and in other places where we tell users to go into Datadog 🙂